sse2neon.h

```diff
@@ -2305,10 +2310,34 @@ FORCE_INLINE __m128 _mm_or_ps(__m128 a, __m128 b)
 
 // Loads one cache line of data from address p to a location closer to the
 // processor. https://msdn.microsoft.com/en-us/library/84szxsww(v=vs.100).aspx
-FORCE_INLINE void _mm_prefetch(const void *p, int i)
+FORCE_INLINE void _mm_prefetch(char const *p, int i)
 {
-    (void) i;
-    __builtin_prefetch(p);
+    switch(i) {
```

Comment:
Run `clang-format -i` for consistent style.

Reply:
I think we should update clang-format to v12, since it is quite hard for mac user install clang-format-12 which doesn't exist on brew. Maybe update to clang-format-14 is a better option.

see https://formulae.brew.sh/formula/clang-format

Reply:
version 12 is the minimal requirement indeed.

Reply:
See https://github.com/DLTcollab/sse2neon/blob/master/CONTRIBUTING.md

Reply:
I was thinking about the `make indent` command, but running for each file in our case is pretty ok, too.